Sources/BraintreeThreeDSecure/BTThreeDSecureRequest.swift

```diff
@@ -7,97 +7,89 @@ import BraintreeCore
+    ///    - customFields: Object where each key is the name of a custom field which has been configured in the Control Panel. In the Control Panel you can configure 3D Secure Rules which trigger on certain values.
+    ///    - dataOnlyRequested:  Optional. Indicates whether to use the data only flow. In this flow, frictionless 3DS is ensured for Mastercard cardholders as the card scheme provides a risk score for the issuer to determine whether to approve. If data only is not supported by the processor, a validation error will be raised. Non-Mastercard cardholders will fallback to a normal 3DS flow.
+    ///    - dfReferenceID: The dfReferenceID for the session. Exposed for testing.
+    ///    - email: Optional. The email used for verification.
+    ///    - exemptionRequested: Optional. If set to true, an exemption to the authentication challenge will be requested.
+    ///    - mobilePhoneNumber: Optional. The mobile phone number used for verification. Only numbers. Remove dashes, parentheses and other characters.
+    ///    - nonce: A nonce to be verified by ThreeDSecure
+    ///    - renderTypes: Optional: List of all the render types that the device supports for displaying specific challenge user interfaces within the 3D Secure challenge. When using `BTThreeDSecureUIType.both` or `BTThreeDSecureUIType.html`, all `BTThreeDSecureRenderType` options must be set. When using `BTThreeDSecureUIType.native`, all `BTThreeDSecureRenderType` options except `.html` must be set.
+    ///    - requestedExemptionType: Optional. The exemption type to be requested. If an exemption is requested and the exemption's conditions are satisfied, then it will be applied.
+    ///    - shippingMethod: Optional. The shipping method chosen for the transaction
+    ///    - uiType: Optional: Sets all UI types that the device supports for displaying specific challenge user interfaces in the 3D Secure challenge. Defaults to `.both`
+    ///    - v2UICustomization: Optional. UI Customization for 3DS2 challenge views.
+    public init(
+        accountType: BTThreeDSecureAccountType = .unspecified,
+        additionalInformation: BTThreeDSecureAdditionalInformation? = nil,
+        amount: NSDecimalNumber? = 0,
+        billingAddress: BTThreeDSecurePostalAddress? = nil,
+        cardAddChallengeRequested: Bool = false,
+        challengeRequested: Bool = false,
+        customFields: [String: String]? = nil,
+        dataOnlyRequested: Bool = false,
+        dfReferenceID: String? = nil,
+        email: String? = nil,
+        exemptionRequested: Bool = false,
+        mobilePhoneNumber: String? = nil,
+        nonce: String? = nil,
+        renderTypes: [BTThreeDSecureRenderType]? = nil,
+        requestedExemptionType: BTThreeDSecureRequestedExemptionType = .unspecified,
+        shippingMethod: BTThreeDSecureShippingMethod = .unspecified,
+        uiType: BTThreeDSecureUIType = .both,
+        v2UICustomization: BTThreeDSecureV2UICustomization? = nil
+    ) {
```

Comment:
note: alphabetized the order of the parameters in the initializer

Reply:
What do you think about grouping the required properties first and then the non-optional properties?